utils/convert.py

```diff
@@ -83,13 +85,13 @@ def __init__(self, spec):
         self.apple = Color.clamp(convert_color(spec, AppleRGBColor))
         self.hsv = convert_color(spec, HSVColor)
 
-    # def __str__(self):
-    #     return ", ".join(["{}"]*4).format(
-    #         self.lab,
-    #         self.hsv,
-    #         self.srgb,
-    #         self.apple,
-    #     )
+    def __str__(self):
+        return "   ".join(["{}"]*4).format(
+            self.lab,
+            self.hsv,
+            self.srgb,
+            self.apple,
+        )
 
     def __repr__(self):
         return "\n".join(["{}"]*4).format(
```

Comment:
Err... why?

That doesn't look like a good __repr__ at all. At least wrap it in some `<Color object ... >` or something. Don't risk anything like `AssertionError: [1] != [1]`.